Source/Public/Clear-BicepModuleCache.ps1

```diff
@@ -0,0 +1,115 @@
+function Clear-BicepModuleCache {
+    [CmdletBinding()]
```

Comment:
To prevent error messages you should set DefaultParameterSet()
Currently you can run this with no parameters set, and get the error `Clear-BicepModuleCache: Parameter set cannot be resolved using the specified named parameters. One or more parameters issued cannot be used together or an insufficient number of parameters were provided.`